spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/dynamodb/DynamoDbAutoConfigurationTest.java

```diff
@@ -61,34 +64,42 @@ void dynamoDBAutoConfigurationIsDisabled() {
 
 	@Test
 	void dynamoDBAutoConfigurationIsEnabled() {
```

Comment:
I think DAX related tests should be extracted into a nested class. The tests we had before - without dax - should remain. This basic tests that verifies that dynamo is enabled should have nothing to do with DAX.